src/logic/Preferences.swift

```diff
@@ -197,6 +197,7 @@ enum ShowOnScreenPreference: String, CaseIterable, MacroPreference {
         switch self {
             case .active: return NSLocalizedString("Active screen", comment: "")
             case .includingMouse: return NSLocalizedString("Screen including mouse", comment: "")
+            case .first return NSLocalizedString("Screen including menu bar", comment: "")
```

Comment:
I think `includingMenubar` would be more descriptive enum name than `first`. Could you change to that?

Reply:
yep, I agree